protocol/communities_messenger_test.go

```diff
@@ -4014,7 +4014,8 @@ func (s *MessengerCommunitiesSuite) TestRequestAndCancelCommunityAdminOffline()
 	s.Require().Empty(aliceJoinedCommunities)
 
 	// pull to make sure it has been saved
-	cancelRequestsToJoin, err := s.alice.MyCanceledRequestToJoinForCommunityID(community.ID())
+	myRequestToJoinId := communities.CalculateRequestID(s.alice.IdentityPublicKeyString(), community.ID())
+	cancelRequestsToJoin, err := s.alice.communitiesManager.GetRequestToJoin(myRequestToJoinId)
```

Comment:
I know you just reused the old name, but it would be clearer like this:
```suggestion
	canceledRequestToJoin, err := s.alice.communitiesManager.GetRequestToJoin(myRequestToJoinId)
```